spring/src/main/java/org/axonframework/spring/config/ApplicationContextLookupParameterResolverFactory.java

```diff
@@ -34,6 +34,7 @@
  * @author Allard Buijze
  * @since 2.1.2
  */
+@Deprecated
```

Comment:
This change warrants an adjustment in the JavaDoc describing why it's deprecated and what (if any) replaces it.